kernel/memory/mapping.c

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: CGL-KFS
+// SPDX-License-Identifier: BSD-3-Clause
+
+/* kernel/memory/mapping.c
+ *
+ *
+ *
+ * created: 2022/12/12 - xlmod <glafond-@student.42.fr>
+ * updated: 2022/12/12 - xlmod <glafond-@student.42.fr>
+ */
+
+#include <stdint.h>
+
+void map_page(void *physaddr, void *virtaddr, uint32_t flags) {
+	uint32_t pagedir_index = (uint32_t)virtaddr >> 22;
+	uint32_t pagetab_index = ((uint32_t)virtaddr >> 12) & (PAGE_TABLE_LENGTH - 1);
```

Comment:
This is an index. I think `uint32_t` is inappropriate here.

I don't like the `pagetab_index` name. Either change it to `pagetable_index` or `pt_index` to be consistent with the following ones

Reply:
this file should not be here, i will remove it.